Corona-Warn-App/build.gradle

```diff
@@ -437,4 +437,7 @@ dependencies {
 
     // ANIMATIONS
     implementation "com.airbnb.android:lottie:3.5.0"
+
+    // HCert
+    implementation("com.augustcellars.cose:cose-java:1.1.0")
```

Comment:
Needs to be added to our internal tool, @AlexanderAlferov can help here.

Reply:
Changed the lib, but still need to add it (where?)

Reply:
I added a task to Jira to ask Katharina (7135)